doc/cephadm/upgrade.rst

```diff
@@ -48,6 +48,27 @@ The automated upgrade process follows Ceph best practices.  For example:
 Starting the upgrade
 ====================
 
+.. note::
+
+   Cephadm by default reduces `max_mds` to `1`. This can be disruptive for large
+   scale CephFS deployments because the cluster cannot easily reduce active MDS(s)
+   to `1`. Therefore, To upgrade MDS(s) without reducing `max_mds`, the
+   `fail_fs` option can to be set to `true` (default value is `false`) prior to
+   initiating the upgrade:
+
+   .. prompt:: bash #
+
+      ceph config set mgr mgr/orchestrator/fail_fs true
+
+   This would:
+               #. Fail CephFS filesystems, bringing active MDS daemon(s) to
+                  `up:standby` state.
+
+               #. Upgrade MDS daemons safely.
+
+               #. Bring CephFS filesystems back up, bringing the state of active
+                  MDS daemon(s) from `up:standby` to `up:active`.
```

Comment:
Add note here about staggered upgrades may be necessary to have access to this new feature.

Reply:
I've added in the beginning of the this note as it would be good to inform before people reach here i.e. end of the note. Do let me know of any change here.